config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-presubmit.yaml

```diff
@@ -1,3 +1,14 @@
+presets:
+- labels:
+    preset-pull-gce-device-plugin-gpu: "true"
+  env:
+  - name: KUBE_FEATURE_GATES
+    value: DevicePlugins=true
+  - name: NODE_ACCELERATORS
+    value: type=nvidia-tesla-k80,count=2
+  - name: CREATE_CUSTOM_NETWORK
```

Comment:
This was for some networking experiment that was done first on GPU tests, I don't think this is needed per se.